weatherreport.cpp

```diff
@@ -71,11 +91,20 @@ namespace WeatherSpace
         string report = Report(sensor);
         assert(report.length() > 0);
     }
+
+    void TestHighPrecipitationLowWind()
+    {
+        HighPrecipitationStub sensor;
+        string report = Report(sensor);
+        cout << "Report: High Precipitation and Low Wind: "<< report << endl;
+        EXPECT_NE(report.find("rain") , string::npos) << "'rain' Expected in report,rather received " << report;
+    }
 }
 
 void testWeatherReport() {
     cout << "\nWeather report test\n";
     WeatherSpace::TestRainy();
     WeatherSpace::TestHighPrecipitation();
```

Comment:
`TestHighPrecipitation` is no longer required, since the function below does the same test?